scenarios/getting_started/go/getting_started.go

```diff
@@ -0,0 +1,110 @@
+package main
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/eclipse/paho.golang/paho"
+	"github.com/joho/godotenv"
+)
+
+type MqttConnectionSettings struct {
+	Hostname  string
+	TcpPort   int
+	UseTls    bool
+	CaFile    string
+	KeepAlive uint16
+	ClientId  string
+	Username  string
+	KeyFile   string
+}
+
+func main() {
+	// Load connection settings
+	godotenv.Load("../.env")
+	cs := MqttConnectionSettings{
```

Comment:
we will repeat the same code in all samples. Can you move this to /mqttclients ?